.github/workflows/ruff-code-checks-and-poetry-cecks.yml

```diff
@@ -0,0 +1,30 @@
+name: Ruff code checks and poetry checks
```

Comment:
You could rename it to something like `Code quality checks` or something less specific imo...